### DISCUSSION TITLE
Validation of FDS and smoke concentration

### DISCUSSION THREAD

**Discussion by Mikkelmhfire** [firemodels/fds#13386] [General]
I've read through the validation guide for FDS and in the summary where all the different quantities are shown in a tabel with the calculated bias, i cant help but notice the bias regarding 1) Smoke concentration = 2,76 and 2) Smoke obscuration = 1,05. 

First of all. Shouldnt these two values follow eachother in bias. I mean if smoke is overpredicted, then the smoke obscuration should also be overpredicted right?

Second. Does the bias for smoke concentration mean that a fire in FDS will produce 2,76 times the amount of an actual fire? And therefor is FDS extremely conservative.

Hopefully someone can clear things up for me. Thanks.

**Comment by drjfloyd** [comment]
The results are based on two different sets experiments with different fuels.  There are some challenges here. 
-In the NIST/NRC cases used for soot density (along with the couple of FM datacenter tests) the actual test measurement was obscuration that was converted to soot density using an assumed extinction coeffiicent. FDS inputs for NIST/NRC use soot yields based on gravimetric measurements.  If you look into the details of these tests, the open door does fairly well but the closed door does not. This suggests that as soot ages the extinction coefficient might change. There are ongoing research efforts in this area that do suggest this might be the case. In the case of soot density, FDS is probably making a reasonable prediction but the data uncertainty is probably higher than we are using in producing the statistics.
-In the FAA tests a different fuel was used with very small fires over short periods of time. This would be akin to the open door tests.

We should probably revisit this section of the Validation Guide to see if other datasets that we already use elsewhere in the guide might give more data to make a judgment on.

Is FDS extremely conservative? Depends. What is the source of your soot yield input? If you are assuming a soot yield or using a soot yield for a small scale flame and applying it to a large fire, then your input is going to be a large driver of predictive error that might be hard to quantify. If you are looking at detection, which typically happens early on at low soot densities, the details of the validation cases suggest that if you have a reasonable soot yield, you are probably doing OK in terms of detection time. Also helping out is for a growing t^2 type fire, soot levels are rapidly increasing and even a factor of 2 error in soot concentration may not be a large change in detection time. If you are looking at long term smoke levels for egress, then the FDS reported visibility may be too high (not only due to the assumed extinction coefficient but also deposition as removal term).

**Comment by Mikkelmhfire** [comment]
Hi drfloyd.

Thanks for your answer. The reason why i'm even asking about the smoke koncentration in the first place, is because i use FDS for fire safety engineering. And almost every single case i have, smoke is the main issue, and often a building is completely filled with smoke even though it has a pretty big fireventilationssystem. Im not saying  that the FDS predictiong is wrong, im just curious why the smoke almost always is such an issue. Thats when i read the 2,76 bias in the validation guide. Ultimately this means that FDS overpredicts a lot right?

For information im using a*t^2 fire with a constant soot_yield of 0,06 g/g. Constant HRRPUA on 1000 kW/m2 and Hc = 20.000 kJ/kg.

**Reply by drjfloyd** [reply]
For typical fuels found in building, the visibility is likely to drive tenability for egress in an atrium smoke management calculation. Over typical egress durations, the ppm of smoke needed to obscure visibility is less than the ppm of CO needed cause to health effects. You should not be judging the adequacy of your solution simply by looking at the smoke 3D results in smokeview. You should be looking at the quantitative outputs of visibility or obscuration and assessing if those still permit wayfinding to exits.  The mere presence of small amounts of smoke does not mean egress is not possible. 

With a ventilation system your predictions are going to be more like the open door tests than the close door tests. There the bias is not as large as when averaged over both the open and close door tests.

![image](https://github.com/user-attachments/assets/b63a0f52-7f30-4f5c-b2fe-ce1e1262fe64)

**Reply by TristanHehnen** [reply]
For a quantitative assessments of the visibility of exist signs and routes, this work may be interesting. It helps generate ASET maps from FDS data that take variations in smoke density, exit sign types and locations, as well as local view obstructions into account.

https://arxiv.org/abs/2404.11439

**Reply by johodges** [reply]
Small addition to Jason's point. The natural ventilation cases with the door closed are particularly challenging to predict smoke concentration because there is higher uncertainty in the HRR in the compartment due to flame extinction. The test series measured HRR with O2 calorimetry and fuel mass loss rate. In the open door cases the HRR measurements from calorimetry were ~14% higher than the ambient fuel flow indicated. This same correction factor was applied in the closed door cases assuming the combustion efficiency was the same in the closed environment.

We also see more mixing within the compartment in FDS than is seen experimentally. You can see a fairly high spread in O2 concentration between the upper layer and lower layer was seen experimentally which is not predicted by FDS. FYI @drjfloyd I noticed the experiment report indicated O2 concentration was reported on a dry basis. The devices from the out repo show the O2 concentration drop to ~14% which is in line with the test plan, but these devices do not use DRY=T. I do not have strong intuition on how much that would change the O2 concentration. 

![image](https://github.com/user-attachments/assets/ccbb1e54-da61-46ac-b7bb-66538ebfabbc)

**Reply by drjfloyd** [reply]
@johodges The NIST/NRC tests did not use O2 calorimetry. The HRR is based on the fuel flow rate which was measured.  The closed door tests were stopped before O2 levels dropped to the point where significant extinction was likely to avoid the potential creation of a flammable gas mixture inside the compartment. 

he DEVC should use DRY in these inputs but at 15 % O (a 25 % drop from ambient) the amount of water vapor is going to be on the order of CO2 mole fraction which was 5 % at end of test.  So not a large change in value for the DEVC.
